pkg/controller/nodeipam/ipam/cloud_cidr_allocator.go

```diff
@@ -40,7 +40,7 @@ import (
 	"k8s.io/kubernetes/pkg/cloudprovider"
 	"k8s.io/kubernetes/pkg/cloudprovider/providers/gce"
 	"k8s.io/kubernetes/pkg/controller"
-	"k8s.io/kubernetes/pkg/controller/node/util"
+	"k8s.io/kubernetes/pkg/controller/nodeutil"
```

Comment:
or maybe invert a little and do `pkg/controller/util/node`? A pattern I've seen elsewhere but YMMV.

Reply:
Done